app/handlers/newflow/educator_complete_profile.rb

```diff
@@ -61,7 +100,7 @@ def push_lead_to_salesforce
     end
 
     def kind_of_educator
-      signup_params.educator_specific_role == 'other' ? signup_params.other_role_name : signup_params.educator_specific_role
+      signup_params.educator_specific_role == OTHER ? signup_params.other_role_name : signup_params.educator_specific_role
```

Comment:
Per your own [recommendation](https://github.com/openstax/accounts/pull/844/files#r431955312) in a code review for my PR (it was a good suggestion indeed 🙂 ), should consider `strip`ping the param:`signup_params.educator_specific_role`. Perhaps you already did, just making sure.